### PR TITLE
add rounding to balance in low credit notification email

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by justinvdm** [comment]
I think we still need to add the change that makes it clear that remaining balance is in credits. Something like: `Your remaining credit balance is 123.45` or `Your remaining balance is 123.45 credits`

**Comment by justinvdm** [comment]
Unless thats for a different PR?

File: go/billing/tasks.py [hunk 1]
```diff
@@ -388,13 +390,14 @@ def create_low_credit_notification(account_number, threshold, balance):
         Decimal(100) - threshold_percent)
     email_from = settings.LOW_CREDIT_NOTIFICATION_EMAIL
     email_to = account.user.email
+    formatted_balance = str(floor(balance*100)/100)
```
Comment (hodgestar) [inline go/billing/tasks.py]: PEP8 violations?

File: go/billing/tests/test_tasks.py [hunk 2]
```diff
@@ -643,7 +643,7 @@ def test_email_sent(self):
                 '70.100'),
             email.subject)
         self.assertTrue('29.900%' in email.body)
-        self.assertTrue('12.34' in email.body)
+        self.assertTrue('12.34.' in email.body)
```
Comment (hodgestar) [inline go/billing/tests/test_tasks.py]: I think this now needs to be `12.34 credits`?

**Comment by hodgestar** [comment]
:+1: (but we should wait for Travis)

**Comment by justinvdm** [comment]
:+1: once travis is happy, looks great

**Pull request merged**